lib/dnsimple/client/registrar.rb

```diff
@@ -67,6 +67,28 @@ def register_domain(account_id, domain_name, attributes, options = {})
         Dnsimple::Response.new(response, Struct::DomainRegistration.new(response["data"]))
       end
 
+      # Retrieves the details of an existing domain registration.
+      #
+      # @see https://developer.dnsimple.com/v2/registrar/#getDomainRegistration
+      #
+      # @example Retrieve the registration 42 for example.com:
+      #   client.registrar.get_domain_registration(1010, "example.com", 42)
+      #
+      # @param  [Integer] account_id the account ID
```

Comment:
@DXTimer just a heads up. The recommended syntax is `parameter [Type] desc`. While also `[Type] parameter desc` is supported, the former is the most evident one, and the one we've been adopting more consistently.

It is possible older code would still adopt the second format. But we should move towards `parameter [Type] desc`.